maintainers/maintainer-list.nix

```diff
@@ -2469,6 +2469,12 @@
     githubId = 30630233;
     name = "Timo Triebensky";
   };
+  birkb = {
+    email = "birk@batchworks.de";
+    github = "birkb";
+    githubId = 10164833;
+    name = "Birk Bohne";
+  };
```

Comment:
please split this into it's own commit called `maintainers: add birkb`

Reply:
done